terraform/api_gateway_domain_name.tf.json.template.py

```diff
@@ -72,7 +72,23 @@
                         "evaluate_target_health": True,
                     }
                 }
-            }
-        } for lambda_name, gateway_id in gateway_ids.items() if gateway_id
+            },
+        },
+         **({
+             "aws_cloudwatch_log_group": {
+                 lambda_name: {
+                     "name": "/aws/apigateway/azul-%s-%s" % (lambda_name, config.deployment_stage),
+                     "retention_in_days": 90,
+                     "provisioner": [{
+                         "local-exec": {
+                             "command": (f"python {config.project_root}/scripts/log_api_gateway.py "
+                                         f"{gateway_id} "
```

Comment:
For appending or prepending a string, we prefer `+` over f-strings.

Reply:
:+1: